source-oracle/replication.go

```diff
@@ -614,7 +622,7 @@ func (s *replicationStream) receiveMessages(ctx context.Context, startSCN, endSC
 	}
 
 	var totalMessages = 0
-	var relevantMessages = 0
+	var fullMessages = 0
```

Comment:
We no longer have "irrelevant" messages from disabled tables, but rather we may have to piece together small messages to compose a larger message which includes all the data, so changing the naming here to reflect that